test/extended/builds/multistage.go

```diff
@@ -79,6 +79,7 @@ COPY --from=test /usr/bin/curl /test/
 
 			s, err := result.Logs()
 			o.Expect(err).NotTo(o.HaveOccurred())
+                        //After bug 1684427 fix, will add more checkpoint info
```

Comment:
Is this PR waiting on another change to merge?

Reply:
Yeah.. I will check step order after bug fix.

Reply:
Bug 1684427 has fixed, remove this comment.